external.bzl

```diff
@@ -426,8 +417,8 @@ def _go_dependencies():
             "@io_bazel_rules_go//third_party:org_golang_x_tools-extras.patch",
             "@io_kythe//third_party/go:add_export_license.patch",
         ],
-        sum = "h1:u+nComwpgIe2VK1OTg8C74VQWda+MuB+wkIEsqFeoxY=",
-        version = "v0.0.0-20191219192050-56b0b28a00f7",
+        sum = "h1:2/D+HOSyyGL4t3L/lSdwKneCXHGwWOIVSTm5SUwY8/E=",
+        version = "v0.0.0-20200219184216-c4d4ea9c797c",
```

Comment:
I think the `build_directives` above need updated so we don't get spurious warnings.